src/Discord.Net.Core/GatewayIntents.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Discord
+{
+    [Flags]
+    public enum GatewayIntents
+    {
+        /// <summary> GUILD_CREATE, GUILD_UPDATE, GUILD_DELETE, GUILD_ROLE_CREATE, GUILD_ROLE_UPDATE, GUILD_ROLE_DELETE, CHANNEL_CREATE, CHANNEL_UPDATE, CHANNEL_DELETE, CHANNEL_PINS_UPDATE </summary>
```

Comment:
I think these might need better summaries.
Maybe something like:
```
/// <summary>
///     This intent includes GUILD_CREATE, GUILD_UPDATE, ...
/// </summary>
```
@Still34 might have a better suggestion.

Reply:
Updated for now, will change out if anyone has other ideas.